generators/app/templates/sources/main/helpers/analytics/analytics.service.ts

```diff
@@ -0,0 +1,76 @@
+import app from 'main.module';
+import {ILogger, LoggerService} from 'helpers/logger/logger';
+import {IApplicationConfig} from 'main.constants';
+
+interface IWindowWithAnalytics extends ng.IWindowService {
+  googleAnalytics: any;
+}
+
+/**
+ * Analytics service: insert Google Analytics library in the page.
+ */
+export class AnalyticsService {
+
+  private logger: ILogger;
+  private analyticsAreActive = false;
+
+  constructor(private $window: IWindowWithAnalytics,
+              private config: IApplicationConfig,
+              logger: LoggerService) {
+
+    this.logger = logger.getLogger('analyticsService');
```

Comment:
Absolutely, logging tracked events was the point. I must have lost it somewhere along the road ; will fix it now.